### PR TITLE
actually call near-vm when using VMKind::NearVm

### PR DESCRIPTION
**Pull request by Ekleog-NEAR**

### CODE REVIEW

File: runtime/near-vm-runner/src/near_vm_runner.rs
Comment (nagisa) [inline runtime/near-vm-runner/src/near_vm_runner.rs:44]: We went over this before, but double-checking these ideally happens before we roll something out (you mentioned believing these should be true, I still haven't double checked it, though.)

File: runtime/near-vm-runner/src/near_vm_runner.rs
Comment (nagisa) [inline runtime/near-vm-runner/src/near_vm_runner.rs:254]: I think we can name this `NearVM`.

File: runtime/near-vm-runner/src/near_vm_runner.rs
Comment (nagisa) [inline runtime/near-vm-runner/src/near_vm_runner.rs:536]: Similarly for this module.

**Review by nagisa — APPROVED** [review]

File: runtime/near-vm-runner/src/near_vm_runner.rs
Comment (Ekleog-NEAR) [reply runtime/near-vm-runner/src/near_vm_runner.rs:536]: So this is a separate crate, that was copy-pasted from wasmer_vm ; if we rename it it’ll collide with the main near_vm crate (that was copy-pasted from the wasmer crate).

So yes, but it’ll have to wait until we merge the near-vm crates all together, as the crate split doesn’t make much sense for us

File: runtime/near-vm-runner/src/near_vm_runner.rs
Comment (Ekleog-NEAR) [reply runtime/near-vm-runner/src/near_vm_runner.rs:44]: This message seems to confirm that these were on-by-default in previous versions of wasmparser: https://github.com/bytecodealliance/wasm-tools/pull/482

**Pull request merged**
